recipe/meta.yaml

```diff
@@ -436,6 +439,9 @@ outputs:
         - sysroot_{{ target_platform }}        # [linux]
         - libgcc-devel_{{ target_platform }}   # [linux]
         - {{ pin_subpackage("clang-" ~ major_version, exact=True) }}
+        - ld64                                 # [osx]
+        - compiler-rt {{ version }}
+        - llvm-openmp {{ version }}
```

Comment:
I'm guessing if we make openmp a dependency of clang, that we could get rid of the symlink to `omp.h` in `llvm-openmp` again?

Reply:
The symlink to `omp.h` is in `clang-21`, not `llvm-openmp`.